lib/openhab/log.rb

```diff
@@ -264,6 +264,7 @@ def level=(level)
     def log_exception(exception)
       exception = clean_backtrace(exception)
       error do
+        name = "In: #{name}\n" if name
```

Comment:
name will never be nil. and the logger name should already be logged as part of the log line. why do you need this?

Reply:
I've removed it instead. As it is, it just makes it harder to read because there's no separation.